x-pack/plugins/ml/public/application/components/data_grid/common.ts

```diff
@@ -415,11 +415,20 @@ export const showDataGridColumnChartErrorMessageToast = (
 // helper function to transform { [key]: [val] } => { [key]: val }
 // for when `fields` is used in es.search since response is always an array of values
 // since response always returns an array of values for each field
-export const getProcessedFields = (originalObj: object) => {
+export const getProcessedFields = (originalObj: object, omitBy?: (key: string) => boolean) => {
```

Comment:
Guess this can be removed now that https://github.com/elastic/kibana/pull/82334 is merged?